target/linux/ipq40xx/patches-5.4/0003-v5.7-ARM-dts-qcom-add-gpio-ranges-property.patch

```diff
@@ -0,0 +1,119 @@
+From 8b99dc0922618062a1589ebd74df6108b4f9ac22 Mon Sep 17 00:00:00 2001
+From: Christian Lamparter <chunkeey@gmail.com>
+Date: Wed, 8 Jan 2020 13:54:55 +0100
+Subject: [PATCH] ARM: dts: qcom: add gpio-ranges property
```

Comment:
Off-topic here, but I wonder whether that one should be moved to generic patches at some point?

Reply:
Since it affects only 2 targets that are in OpenWrt I don't know if it makes sense.
It's gonna be dropped at the next kernel update anyway.